SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKSimulatedPlayerDeployment.cpp

```diff
@@ -344,6 +400,49 @@ void SSpatialGDKSimulatedPlayerDeployment::OnPrimaryLaunchConfigPathPicked(const
 	SpatialGDKCloudLauncherSettings->SetPrimaryLaunchConfigPath(PickedPath);
 }
 
+TSharedRef<SWidget> SSpatialGDKSimulatedPlayerDeployment::OnGetPrimaryDeploymentRegionCode()
+{
+	FMenuBuilder MenuBuilder(true, NULL);
+	UEnum* pEnum = FindObject<UEnum>(ANY_PACKAGE, TEXT("ERegionCode"), true);
+
+	for (int32 i = 0; i < pEnum->NumEnums() - 1; i++)
+	{
+		int64 CurrentEnumValue = pEnum->GetValueByIndex(i);
+		FUIAction ItemAction(FExecuteAction::CreateSP(this, &SSpatialGDKSimulatedPlayerDeployment::OnPrimaryDeploymentRegionCodePicked, CurrentEnumValue));
+		MenuBuilder.AddMenuEntry(pEnum->GetEnumTextByValue(CurrentEnumValue), TAttribute<FText>(), FSlateIcon(), ItemAction);
+	}
+
+	return MenuBuilder.MakeWidget();
+}
+
+TSharedRef<SWidget> SSpatialGDKSimulatedPlayerDeployment::OnGetSimulatedPlayerDeploymentRegionCode()
+{
+	FMenuBuilder MenuBuilder(true, NULL);
+	UEnum* pEnum = FindObject<UEnum>(ANY_PACKAGE, TEXT("ERegionCode"), true);
+
```

Comment:
Same here, should we check if enum is valid?

Reply:
Good catch - will update the code 👍